node/cmd/ccq/metrics.go

```diff
@@ -108,4 +111,19 @@ var (
 			Name: "ccq_server_total_number_of_successful_reconnects",
 			Help: "Total number of successful reconnects to bootstrap peers",
 		})
+
+	maxConcurrentQueriesByChain = promauto.NewGaugeVec(
```

Comment:
I wonder if it's also worth tracking `currentConcurrentQueriesByChain` to see how bursty the overlapping requests are.